src/url.rs

```diff
@@ -79,6 +79,17 @@ pub fn parse_data_url(url: &Url) -> (String, String, Vec<u8>) {
     (media_type, charset, blob)
 }
 
+pub fn referer_url(url: Url) -> Url {
+    let mut url = url.clone();
+    // https://httpwg.org/specs/rfc9110.html#field.referer
```

Comment:
I think putting "Spec: " or "Reference: " before the URL could be a good way to make it more readable, otherwise it's just a commented out URL.  I try not to put them myself, since URLs tend to become 404/301 sometimes, and then it'll have to be updated.

There's this file here https://github.com/Y2Z/monolith/blob/master/docs/references.md that I created to keep all used references, maybe we could move it there, what do you think?

Reply:
Just add a `Spec: ` prefix. 

It's nice to have internal documentation and refer to it, but in case when ref leads just to another/external ref, in my vision, it doesn't make sense. Anyway in case external link "died" one have to update it no matter its place in the source code.